src/components/episode-content.vue

```diff
@@ -34,11 +34,15 @@ if (episode)
   <div v-if="episode" container mx-auto px-4 mt-5>
     <div mb-5>
       <h1 text-center text-2xl font-bold mb-2 v-html="episode.title" />
-      <div flex justify-center my-2>
-        <a :href="episode.link" target="_blank" text-black dark:text-white>
-          <div i-fluent:headphones-sound-wave-24-filled text-xl />
-        </a>
-      </div>
+      <figure box-border my-2>
+        <audio
+          controls
+          :src="episode.link"
+          style="width: 100%; min-width: 300px"
+        >
+          Your browser does not support the <code>audio</code> element.
+        </audio>
+      </figure>
```

Comment:
I'm wondering if you could add a condition on here. Only show the audio element when the `episode.link` ends with mp3 or m4a

Reply:
Done